apps/sensenet/src/components/dialogs/add.tsx

```diff
@@ -1,15 +1,15 @@
 import DialogContent from '@material-ui/core/DialogContent'
 import DialogTitle from '@material-ui/core/DialogTitle'
 import { NewView } from '@sensenet/controls-react'
-import { GenericContent, Schema } from '@sensenet/default-content-types'
+import { Schema } from '@sensenet/default-content-types'
 import React from 'react'
 import { useLogger, useRepository } from '@sensenet/hooks-react'
 import { useLocalization } from '../../hooks'
 import { useDialog } from './dialog-provider'
 
 export type AddDialogProps = {
   schema: Schema
-  parent: GenericContent
+  parent: string
```

Comment:
I think it would make sense to rename this property to parentPath then.

Reply:
done